gammapy/cube/tests/test_psf_kernel.py

```diff
@@ -49,3 +49,15 @@ def test_psf_kernel_from_gauss_read_write(tmpdir):
     kernel.write(filename, overwrite=True)
     newkernel = PSFKernel.read(filename)
     assert_allclose(kernel.psf_kernel_map.data, newkernel.psf_kernel_map.data)
+
+def test_make_image():
+    energy_axis = MapAxis.from_edges([0.1, 0.5, 2.5,  10], name="energy", unit="TeV", interp="log")
+    geom = WcsGeom.create(binsz=0.1, npix=150, axes=[energy_axis])
+    exposure = Map.from_geom(geom)
+    exposure.data = np.ones(exposure.data.shape)
+    sigma = 0.5 * u.deg
+    kernel = PSFKernel.from_gauss(geom, sigma)
```

Comment:
Here you should probably have two different sigmas and average over bins such that the expected statistic is constant, e.g. for constant exposure and E^-2 spectrum 100, 150 and 230 GeV